snapcast-client/Dockerfile.aarch64

```diff
@@ -0,0 +1,14 @@
+FROM balenalib/raspberrypi3-debian
+
+ARG SNAPCAST_VERSION=0.17.1
```

Comment:
Is this a duplicate of the one in the compose file?

Reply:
I added this as a default in case you don't provide it via the compose file.